OracleGoldenGate/README.md

```diff
@@ -0,0 +1,98 @@
+Oracle GoldenGate on Docker
+===============
+Sample Docker build files to provide an installation of Oracle GoldenGate for DevOps users. For more information about Oracle GoldenGate please see the [Oracle GoldenGate On-line Documentation](https://docs.oracle.com/goldengate/c1221/gg-winux/index.html).
+
+## How to build and run
+This project provides a Dockerfile tested with:
+ * Oracle GoldenGate 12c Release 2 (12.2.0.1)
+ * Oracle GoldenGate Standard Edition 12c Release 3 (12.3.0.1)
+ * Oracle GoldenGate Microservices Architecture 12c Release 3 (12.3.0.1)
+
+To build the images, use the [buildDockerImage.sh](buildDockerImage.sh) script.
+
+### Building Oracle GoldenGate Docker Images
+**IMPORTANT:** You must download the installation binaries of Oracle GoldenGate. You only need to provide the binaries for the version you plan to install. The binaries can be downloaded from the [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html). You also must have Internet connectivity when building the Docker image for the package manager. Note that you must not uncompress the Oracle GoldenGate binaries. The script will handle that for you and fail if you uncompress them manually!
+
+Once you have downloaded the Oracle GoldenGate software, run the **`buildDockerImage.sh`** script:
+
+    [oracle@localhost dockerfiles]$ ./buildDockerImage.sh -h
+    Oracle GoldenGate distribution ZIP file not specified.
+
+    Usage: ./buildDockerImage.sh [-h | <ogg-zip-file-name>] [<docker-build-options> ...]
+    Where:
+      ogg-zip-file-name       Name of OGG ZIP file
+      docker-build-options    Command line options for Docker build
+
+    Example:
+      buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip --no-cache
+
+**IMPORTANT:** The result will be a Docker image with the Oracle GoldenGate binaries installed. Created images will follow the naming convention of **oracle/goldengate-&lt;edition&gt;:&lt;version&gt;**, for example:
+
+- `oracle/goldengate-standard:12.2.0.1.1`
+- `oracle/goldengate-standard:12.3.0.1.0`
+- `oracle/goldengate-microservices:12.3.0.1.0`
+
+#### Changing the Base Image
+By default, the base image used when building Oracle GoldenGate Docker images is `container-registry.oracle.com/database/instantclient:12.2.0.1`.  This value can be changed by setting the environment variable `BASE_IMAGE` when executing the `buildDockerImage.sh` script. For example:
+
+    BASE_IMAGE="container-registry.oracle.com/database/enterprise" ./buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip
```

Comment:
Also, you don't explain _why_ or _when_ a user might want to change the base image.